### PR TITLE
Add WPE #0000

### PR DESCRIPTION
**Pull request by ram-nad**
The purpose of this WPE is to establish WPEs as the primary method for proposing changes to the WildfirePy project.

### CODE REVIEW

File: WPE_0000.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+
+Contributors
+------------
+
+Raahul Singh (@Raahul-Singh)
+
+
+Previous Versions
+-----------------
```
Comment (Raahul-Singh) [inline WPE_0000.rst:121]: We don't need this.

**Comment by ram-nad** [comment]
@masterashu @abhijeetmanhas @AkshobhyaP @kakirastern @Raahul-Singh @siddhantsiddhant @stoic-signs @FieryBlade-313

File: WPE_0000.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+References
+----------
+
+This WPE is inspired by `EinsteinPy's EPEs <https://google.com>`_.
+
+
+Decision rationale
+------------------
+
```
Comment (Raahul-Singh) [inline WPE_0000.rst:112]: This can be removed.

File: WPE_0000.rst (hunk 1 above)
Comment (ram-nad) [reply WPE_0000.rst:121]: Okk

Comment (Raahul-Singh) [inline WPE_0000.rst:121]:
```suggestion

Previous Versions
-----------------

This is the first version of this document.
```

File: WPE_0000.rst (hunk 2 above)
Comment (ram-nad) [reply WPE_0000.rst:112]: We will just before this WPE is accepted/rejected.

File: WPE_0000.rst [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+Also, each WPE will have a DOI badge that needs to be generated and added to
+the `README.md` of the repository before PR is finally merged. The badge needs
+to be regenerated for updated WPEs.
+
+
+References
+----------
+
+This WPE is inspired by `EinsteinPy's EPEs <https://google.com>`_.
```
[the remark this answers is not in the document]
Comment (ram-nad) [reply WPE_0000.rst:107]: Sorry. Right.

[the remark this answers is not in the document]
Comment (Raahul-Singh) [reply WPE_0000.rst:107]: Good find @abhijeetmanhas

**Pull request merged**